react/week1/todo-app/src/App.js

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+import TodoList from "./List"
+
+function App() {
+  const Todo = ({ title, deadLine }) => {
```

Comment:
hi, good job. but  this function should be out of App function

Reply:
Ok thank you, I will fix that :)